src/domain/notifier.py

```diff
@@ -0,0 +1,32 @@
+"""Notification module"""
+
+import logging
+from abc import abstractmethod
+from enum import Enum
+
+logger = logging.getLogger(__name__)
+
+
+class Notifier:
+    """Base class for notifiers."""
+
+    class NotifierTypes(Enum):
+        Email = "Email"
```

Comment:
Email is not a class. Since it's a regular class member, its name should be all in lower case (with underscores, if needed).

Reply:
I guess enum data is considered constant as i found many examples in uppercase. I am ok either way, but since this is also the case for other enums let's define our guideline and then we will change them all.

Reply:
Right. Then uppercase is fine.

Reply:
Ok, will do this refactor as separate step to cover all cases around the code.